qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +92,17 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_negative:
+            type = nvdimm_negative
+            start_vm = no
+            size_mem = 8G
+            pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
+            variants:
+                - align_max:
+                    align_mem = 4G
+                    error_msg = "the alignment \(0x[0-9A-Fa-f]+\) is not supported|the alignment \(0x[0-9A-Fa-f]+\) exceeds the expected maximum alignment"
```

Comment:
The align_mem `4G` already fixed, why don't use the `0x100000000` directly ? @huangyum

Reply:
I'm not sure if the max value would change in the future, so use an exp.

Reply:
> I'm not sure if the max value would change in the future, so use an exp.

It does not matter as my understanding.  If the max value be bigger than 4G in the feature, the `align_mem` still need be updated.  The value in the error_msg is match with the setting of `align_mem`.

Reply:
Make sense. Updated. Thanks.